packages/epo-react-lib/src/styles/base/index.ts

```diff
@@ -0,0 +1,27 @@
+import { css } from "styled-components";
+import { aButton, aButtonTheme } from "@/styles/mixins/appearance";
+
+export default css`
```

Comment:
grab a couple of the base styles from `rubin-obs-client` and add them to `GlobalStyles`